woocommerce/wc-admin
# Update all wc-admin endpoints to v4

Fixes #1264 

Bumps all wc-admin endpoints to v4 and hooks up the orders page.

### Screenshots
<img width="1111" alt="screen shot 2019-01-16 at 5 39 38 pm" src="https://user-images.githubusercontent.com/10561050/51239977-b81a0a00-19b5-11e9-8050-b0bf5d170fe2.png">

### Detailed test instructions:

1.  Go through each wc-admin page. 
2.  Make sure pages are loading without any API 404 errors and data still loads correctly.
3.  Check that the Orders report loads the correct data with the new orders endpoint hooked up.
4.  Check that orders report contains the correct coupon data.
5.  Make sure all tests are passing.
